doc/classes/TabBar.xml

```diff
@@ -223,6 +239,10 @@
 		<member name="drag_to_rearrange_enabled" type="bool" setter="set_drag_to_rearrange_enabled" getter="get_drag_to_rearrange_enabled" default="false">
 			If [code]true[/code], tabs can be rearranged with mouse drag.
 		</member>
+		<member name="focus_mode" type="int" setter="set_focus_mode" getter="get_focus_mode" overrides="Control" enum="Control.FocusMode" default="2" />
+		<member name="input_rollover" type="bool" setter="set_input_rollover" getter="get_input_rollover" default="false">
+			If [code]true[/code], pressing [code]ui_right[/code] on the last tab will select the first tab and pressing [code]ui_left[/code] on the first tab will select the last one.
+		</member>
```

Comment:
I wonder if this option is even necessary, as the rollover behavior is present in other nodes without a way to toggle it off, and no one is complaining.

Reply:
I'm on the fence of this one, for me both approaches are fine. However I can think of one case where enabled rollover could cause issues: 
![button_next_to_tabs](https://github.com/godotengine/godot/assets/69091976/33d5d898-acf1-4fd0-8864-42a4cc90064a)
If there's a button next to tabs it's possible to move the focus from button to tabs using arrows, but it won't be possible to move focus from tabs back to button if rollover is enabled.
Alternative solution would be to not allow rollover at all

Reply:
Hi, I did not remove the option yet as I'm not sure what's the final decision for keeping or removing the option or if we should keep rollover behavior at all.